viaforge-mc1122/src/main/resources/mcmod.info

```diff
@@ -4,15 +4,12 @@
     "name": "ViaForge",
     "description": "Client-side Implementation of the Via* projects for Minecraft Forge",
     "version": "${version}",
-    "mcversion": "[1.12.2]",
+    "mcversion": "[1.12,)",
```

Comment:
Did you try if loading on 1.12.0 and 1.12.1 Forge works?

Reply:
yes